cmd/veeam-sos-api.go

```diff
@@ -118,8 +120,8 @@ func veeamSOSAPIGetObject(ctx context.Context, bucket, object string, rs *HTTPRa
 	switch object {
 	case systemXMLObject:
 		si := systemInfo{
-			ProtocolVersion: "1.0",
-			ModelName:       "MinIO " + ReleaseTag,
+			ProtocolVersion: `"1.0"`,
+			ModelName:       "\"MinIO \"" + ReleaseTag + "\"",
```

Comment:
There are three double quotes here, it doesn't look right

Reply:
Yeah it's fixed.